sentry/src/main/java/io/sentry/instrumentation/file/FileInputStreamInitData.java

```diff
@@ -0,0 +1,21 @@
+package io.sentry.instrumentation.file;
+
+import io.sentry.ISpan;
+import java.io.File;
+import java.io.FileInputStream;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+final class FileInputStreamInitData {
+
+  @Nullable File file;
+  @Nullable ISpan span;
+  @NotNull FileInputStream delegate;
+
+  public FileInputStreamInitData(@Nullable File file, @Nullable ISpan span,
```

Comment:
always mark fields as `final` to make them immutable

eg `var` and `val` on Kotlin